atom/browser/api/atom_api_extension.cc

```diff
@@ -220,6 +222,33 @@ void Extension::Enable(const std::string& extension_id) {
 }
 
 // static
+void Extension::Remove(const std::string& extension_id) {
+  const extensions::Extension* extension =
+    GetInstance()->extensions_.GetByID(extension_id);
+  if (extension) {
```

Comment:
Isn't NOTIFICATION_DISABLE_USER_EXTENSION_REQUEST handled in AtomExtensionSystem to disable?